.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,41 @@
+---
+name: 📚 Documentation
+about: Add/Improve Documentation
+title: 📚 <Insert Title>
+labels: documentation
+assignees: ''
+
+---
+
+# Summary
+<!---
+A short summary on what of what needs to be documented or updated.
+--->
+
+# Details
+<!---
+A detailed description about what kind of documentation should be added and/or updated and why.
+--->
+
+## Background & Context
+<!---
+Why should this documentation be added/updated?
```

Comment:
This interferes with previous section, also asking to specify a rationale ("why").

Proposal: Could we agree on only  a single place to describe a rationale?